src/main/java/frc/robot/commands/AssistedBalanceCommand.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.SwerveSubsystem;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An AssistedBalaceCommand command that uses SwerveSubsystem and SimulationSubsystemd */
+public class AssistedBalanceCommand extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final SwerveSubsystem swerve_subsystem;
+  PIDController pidController = new PIDController(0.2, 0, 0);
```

Comment:
Make private

Add TunableNumbers for the PID constants.

Reply:
I think this is done but I'm going to ask at next meeting so I am not resolving this change yet.